src/app/editor/_components/editorLeftSideBar/EditorLeftSideBar.tsx

```diff
@@ -0,0 +1,189 @@
+          gap: "14px",
+        }}
+      >
+        <Typography
+          sx={{
+            fontWeight: "bold",
+            fontSize: "16px",
+          }}
+        >
+          Iterations
+        </Typography>
+
+        <Button
+          sx={{
+            display: "flex",
+            justifyContent: "start",
+            alignItems: "center",
+            gap: "4px",
+            width: "100%",
+            backgroundColor: "primary.main",
+            minHeight: "auto",
+            color: "white",
+            py: "10px",
+          }}
+        >
+          <AddIcon />
+          <Typography
+            sx={{
+              fontSize: "15px",
+            }}
+          >
+            {STATIC_TEXTS.EDITOR_PAGE.ADD_ITERATION}
+          </Typography>
+        </Button>
+        <Button
+          sx={{
+            display: "flex",
+            justifyContent: "start",
+            alignItems: "center",
+            gap: "4px",
+            width: "100%",
+            backgroundColor: "secondary.main",
+            minHeight: "auto",
+            py: "10px",
+          }}
+        >
+          <InsertPhotoOutlinedIcon />
+          <Typography
+            sx={{
+              fontSize: "15px",
+              color: "primary.main",
+            }}
+          >
+            {STATIC_TEXTS.EDITOR_PAGE.IMPORT_TEXT}
+          </Typography>
+        </Button>
+        <Box
+          sx={{
+            color: "black",
+            width: "100%",
+          }}
+        >
+          <Box
+            sx={{
+              borderRadius: "4px",
+              border: "1px solid ",
+              borderColor: "common.strokePrimary",
+              padding: "10px",
+              marginTop: "8px",
+              display: "flex",
+              justifyContent: "space-between",
+              alignItems: "center",
+            }}
+          >
+            <Typography
+              sx={{
+                fontSize: "16px",
+                color: "font.secondary",
+              }}
+            >{`Iteration 0`}</Typography>
+            <ArrowForwardIosOutlinedIcon
+              sx={{
+                color: "common.gray",
+                fontSize: "12px",
+              }}
+            />
+          </Box>
+          {iterations?.map((item, index) => {
+            return (
+              <Box
+                sx={{
+                  borderRadius: "4px",
+                  border: "1px solid ",
+                  borderColor: "common.strokePrimary",
+                  padding: "10px",
+                  marginTop: "8px",
+                  display: "flex",
+                  justifyContent: "space-between",
+                  alignItems: "center",
+                }}
+                key={index + 1}
+              >
+                <Typography
+                  sx={{
+                    fontSize: "16px",
+                    color: "font.secondary",
+                  }}
+                >{`Iteration ${index + 1}`}</Typography>
```

Comment:
iteration should be an array of objects where user can provide the name of iteration. If name not provided then we should show iteration with index